build.sbt

```diff
@@ -40,12 +40,12 @@ lazy val root = Project("spark-redshift", file("."))
   .settings(
     name := "spark-redshift",
     organization := "io.github.spark-redshift-community",
-    scalaVersion := "2.11.12",
+    scalaVersion := "2.12.11",
     crossScalaVersions := Seq(
       "2.11.12",
       "2.12.11"
     ),
```

Comment:
thanks, I will remove it

Reply:
Why removing it? Can there be people using scala 2.12 and spark 3 ?

Reply:
@lucagiovagnoli this is a good point since spark 3 will no longer support scala 2.11. So maybe we should have a new release branch for spark 3 compatible?

Reply:
Yeah, can remove cross publish because scala 2.11 isn't compatible with spark 3.  Will only be publishing scala 2.12 versions.

Reply:
Oh right yes. So I'd be in favor of a separate release branch for spark 3, but I'd be worried about maintenance overhead. Maybe we can have master "follow" spark 3 and still accept PRs on a branch pointing to the latest commit using spark 2.x